src/Surfnet/StepupSelfService/SelfServiceBundle/Resources/views/Registration/Sms/sendChallenge.html.twig

```diff
@@ -25,17 +25,29 @@
     {% endif %}
 
     {{ form_start(form)|replace({'form-horizontal': ''})|raw }}
+    {{ form_errors(form) }}
```

Comment:
And if you did this because previously it would create incorrect markup or something, then [this change](https://github.com/SURFnet/Stepup-RA/pull/111/commits/933beb755da14135b6fc44c24ec7c87709b17ef3#diff-360787ad8c854d42e4ff551308652b0bR25) may be incorrect.